src/app.ts

```diff
@@ -11,6 +11,7 @@ export type AppCreateOpts = {
   mountTo: HTMLElement;
   overlayButtons: HTMLDivElement;
   initialState: Partial<AppState>;
+  useWebGL?: boolean;
```

Comment:
You should expect to always know the naswer to this by the time you create the app, so this shouldn't be optional